.github/workflows/ci.yml

```
@@ -80,134 +80,152 @@ jobs:
     name: Build docs
     runs-on: ubuntu-latest
     env:
-      TRAVIS_JOB_NAME: "Build documentation"
       PYTHON_VERSION: 3.6
+      CI_JOB_TYPE: "Documentation"
     steps:
       - uses: actions/checkout@master
-      - name: "Build documentation"
+      - name: "Build CI image"
+        run: ./scripts/ci/ci_prepare_image_on_ci.sh
+      - name: "Build docs"
         run: ./scripts/ci/ci_docs.sh
 
   tests-p36-postgres-integrations:
-    name: "Tests [Postgres9.6][Py3.6][integrations]"
+    name: "[Pg9.6][Py3.6][integrations]"
     runs-on: ubuntu-latest
     needs: [statics, statics-tests]
     env:
-      TRAVIS_JOB_NAME: "Tests [Postgres9.6][Py3.6][integrations]"
       BACKEND: postgres
       PYTHON_VERSION: 3.6
       POSTGRES_VERSION: 9.6
       ENABLED_INTEGRATIONS: "cassandra kerberos mongo openldap rabbitmq redis"
       RUN_INTEGRATION_TESTS: all
+      CI_JOB_TYPE: "Tests"
     steps:
       - uses: actions/checkout@master
-      - name: "Tests [Postgres9.6][Py3.6][integrations]"
+      - name: "Build CI image"
+        run: ./scripts/ci/ci_prepare_image_on_ci.sh
+      - name: "Tests"
         run: ./scripts/ci/ci_run_airflow_testing.sh
 
   tests-p36-postgres-providers:
-    name: "Tests [Postgres10][Py3.6][providers]"
+    name: "[Pg10][Py3.6][prov]"
     runs-on: ubuntu-latest
     needs: [statics, statics-tests]
     env:
-      TRAVIS_JOB_NAME: "Tests [Postgres10][Py3.6][providers]"
       BACKEND: postgres
       POSTGRES_VERSION: 10
       PYTHON_VERSION: 3.6
+      CI_JOB_TYPE: "Tests"
     steps:
       - uses: actions/checkout@master
-      - name: "Tests [Postgres10][Py3.6][providers]"
+      - name: "Build CI image"
+        run: ./scripts/ci/ci_prepare_image_on_ci.sh
+      - name: "Tests"
         run: ./scripts/ci/ci_run_airflow_testing.sh tests/providers
 
   tests-p36-postgres-core:
-    name: "Tests [Postgres9.6][Py3.6][core]"
+    name: "[Pg9.6][Py3.6][core]"
     runs-on: ubuntu-latest
     needs: [statics, statics-tests]
     env:
-      TRAVIS_JOB_NAME: "Tests [Postgres9.6][Py3.6][core]"
       BACKEND: postgres
       POSTGRES_VERSION: 9.6
       PYTHON_VERSION: 3.6
+      CI_JOB_TYPE: "Tests"
     steps:
       - uses: actions/checkout@master
-      - name: "Tests [Postgres9.6][Py3.6][core]"
+      - name: "Build CI image"
+        run: ./scripts/ci/ci_prepare_image_on_ci.sh
+      - name: "Tests"
         run: ./scripts/ci/ci_run_airflow_testing.sh --ignore=tests/providers
 
 
   tests-p37-sqlite-integrations:
-    name: "Tests [Sqlite][3.7][integrations]"
+    name: "[Sqlt][3.7][int]"
```

```suggestion
    name: "[Sqlite][3.7][int]"
```
please.

It's not that long and i've never seen Sqlite abbreviated before.

---

OK OK